packages/gallery/src/components/gallery/proGallery/galleryView.js

```diff
@@ -128,6 +130,30 @@ class GalleryView extends React.Component {
     }
   }
 
+  onKeyPress(e) {
```

Comment:
it is  function specific to load more button so it should reflect that and it should also be clrear what event this is in the function name - onLoadMoreButtonKeyDown